runtime/onert/backend/cpu/ops/PadLayer.cc

```diff
@@ -55,11 +54,11 @@ void PadLayer::run()
 {
   if (_input->data_type() == OperandType::FLOAT32)
   {
-    padFloat32();
+    PadImpl<float, float>(_constantValueData.f);
```

Comment:
I wonder whether we strictly need two typenames here? The padded constant values are likely to be of the same datatype as the input and output tensors.

Reply:
As @dr-venkman commented, input, output and constant values are same datatype. Tensorflow lite kernel check type for these

```cpp
  PadContext op_context(context, node);
  TF_LITE_ENSURE_TYPES_EQ(context, op_context.input->type,
                          op_context.output->type);
  if (op_context.constant_values != nullptr) {
    TF_LITE_ENSURE_TYPES_EQ(context, op_context.input->type,
                            op_context.constant_values->type);
  }
```

Reply:
-> I wonder whether we strictly need two typenames here. The padded constant values are likely to be of the same datatype as the input and output tensors.

I agree.
I fixed it.